src/fides/api/service/connectors/saas_connector.py

```diff
@@ -104,25 +105,28 @@ def query_config(self, node: ExecutionNode) -> SaaSQueryConfig:
     def get_client_config(self) -> ClientConfig:
         """Utility method for getting client config according to the current class state"""
         saas_config_client_config = self.saas_config.client_config
-        required_current_saas_request = self.current_saas_request
-        assert required_current_saas_request is not None
```

Comment:
I needed to get rid of this assert so I can call `create_client` with no issue from fidesplus https://github.com/ethyca/fidesplus/pull/1524/files#diff-b41570e64a5a6483a62b6defd81e4cedba6f8ad778923245deb2060b7de008d3R71. We won't have a `current_saas_request` when calling this from fidesplus.